<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventStatePropEx</class>
 <widget class="QWidget" name="EventStatePropEx">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>183</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#ConditionStatePropEx{
background:#35363C;
}
*{
background:transparent;
font-size:14px;
color:white;
}
QSpinBox{
border:none;
border-radius:4px;
padding-left:12px;
}
QSpinBox:hover{
background:#4C4E56;
}
QSpinBox::up-button {
    subcontrol-origin: border;
    subcontrol-position: top right; /* position at the top right corner */

    width: 16px; /* 16 + 2*1px border-width = 15px padding + 3px parent border */
    /*border-image: url(:/res/arrow_up.svg) 1;*/
    border-width: 1px;
}

QSpinBox::up-arrow {
    image: url(:/res/arrow_up.svg);
    width: 12px;
    height: 12px;
}

QSpinBox::down-button {
    subcontrol-origin: border;
    subcontrol-position: bottom right; /* position at bottom right corner */

    width: 16px;
    /*border-image: url(:/res/arrow_down.svg) 1;*/
    border-width: 1px;
    border-top-width: 0;
}

QSpinBox::down-arrow {
    image: url(:/res/arrow_down.svg);
    width:12px;
    height: 12px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>12</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">font-size:17px;font-weight:bold;</string>
     </property>
     <property name="text">
      <string>事件触发状态属性</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">QLineEdit{
border:none;
border-bottom:1px solid #5A636C;
padding-left:8px;
padding-right:8px;
}
QLineEdit:hover{
background:#4C4E56;
}
QLineEdit:focus{
border:1px solid #02BFFF;
}
QLabel[mstyle=&quot;hint&quot;]{
font-size:12px;color:#A0C2D3;
}</string>
     </property>
     <layout class="QGridLayout" name="event_prop_edit_layout" columnstretch="0,0">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <property name="horizontalSpacing">
       <number>8</number>
      </property>
      <property name="verticalSpacing">
       <number>4</number>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>重试次数：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>正常触发信号：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="input_retry_callback">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="input_retry">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>超时时间：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>重试回调函数：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="input_timeout">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="suffix">
         <string> ms</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>异常触发信号：</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../app.qrc"/>
 </resources>
 <connections/>
</ui>
